universal-login-react/test/core/services/DisconnectAccountService.test.ts

```diff
@@ -24,6 +25,7 @@ describe('DisconnectAccountService', () => {
   before(async () => {
     ([wallet] = getWallets(createMockProvider()));
     ({deployedWallet, relayer} = await setupDeployedWallet(wallet, ensName));
+    (deployedWallet.sdk.provider as any).pollingInterval = 100;
```

Comment:
why not 5?

Reply:
This test will fail with 5 or 10 with timeout